kore/src/Kore/Attribute/Axiom.hs

```diff
@@ -281,9 +282,32 @@ mapAxiomVariables adj axiom@Axiom { concrete, symbolic } =
         }
 
 getPriorityOfAxiom
-    :: forall attrs. From attrs (Priority, Owise) => attrs -> Integer
-getPriorityOfAxiom attrs
-  | isOwise   = owisePriority
-  | otherwise = fromMaybe defaultPriority getPriority
-  where
-    (Priority { getPriority }, Owise { isOwise }) = from @attrs attrs
+    :: forall attrs
+    .  HasCallStack
+    => From attrs (Priority, Owise, Simplification)
```

Comment:
We should wrap this combination of `(Priority, Owise, Simplification)` in a `newtype`, but I'm not sure what to call it.